encoders/numeric/RandomGaussianEncoder/tests/test_randomgaussianencoderload.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import pickle
+
+from .. import RandomGaussianEncoder
+from .. import TransformEncoder
+
+
+class RandomGaussianEncoderLoadTestCase:
```

Comment:
Please adapt it to `pytest` standard.

this means.
No class, only a set of `test_....` functions.
Be aware of temporary files that need to be removed (see AnnoyIndexer tests under hub for reference)